src/artemis/ispyb/store_in_ispyb.py

```diff
@@ -119,7 +119,7 @@ def _store_data_collection_table(self, data_collection_group_id: int) -> int:
         params["xbeam"], params["ybeam"] = beam_position
         params["xtal_snapshot1"], params["xtal_snapshot2"], params["xtal_snapshot3"] = [
             self.ispyb_params.xtal_snapshots
-        ] * 3
+        ]
```

Comment:
I thought we passed this in as a list of length 3 so we didn't need the square brackets?